### ISSUE TITLE
Colab implementation

### ISSUE THREAD

**Issue by athoma28** [jiah-cloud/Align3R#3]
Hi! I'm excited to try out your research for myself! I was wondering if you might be able to provide a Colab notebook for inference. I spent a while trying to get one to work, but had trouble and eventually got too far in the weeds for it to work. I'm not clear about where the models should be placed, or how to get the dust3r module to be accessible from the PATH in Colab. Maybe someone else could figure it out?

**Comment by jiah-cloud** [comment]
Thank you for recognizing our work! Recently, we discovered and fixed some bugs in our code. You can now pull the updated version. Additionally, we have provided a demo.sh script, which allows you to test our model on any video.
![3779af3f759a8b126608182568566199](https://github.com/user-attachments/assets/24458ddc-fccc-49d4-a2db-ddd7d991b01a)
If you have any other questions or issues, please list them in detail. We greatly appreciate your feedback.

**Comment by athoma28** [comment]
Yes, I was trying to use the demo.sh script, but it wasn't able to access dust3r as a module the way I was doing it in Colab. Can you add in the readme where the model files should all be placed in the folder structure? Thank you for your fast response!

**Comment by jiah-cloud** [comment]
We think the issue might be related to the line in our scripts where we add `export PYTHONPATH=$PYTHONPATH:Align3R`, which redirects the system path. One way to solve the problem of not being able to access `dust3r` as a module in Colab is by using the following approach:

```python
import sys
sys.path.append('path_to_repo')
```
